apps/admin/components/User/UserList.tsx

```diff
@@ -40,25 +64,165 @@ export const UserList: FC<Props> = (props) => {
       {users.map((user) => (
         <tr key={user.id}>
           <td>{user.id}</td>
-          <td>
-            <div className="position-relative mx-auto" style={{width: '70px', height: '70px'}}>
-              <img
-                style={{objectFit: 'contain', width: '70px', height: '70px'}}
-                alt={user.firstName}
-                src={user.imageUrl}
-              />
-            </div>
-          </td>
-          <td>{user.firstName + user.lastName}</td>
-          <td>{user.email}</td>
-          <td>{user.age}</td>
-          <td>{user.city}</td>
-          <td>{user.degree}</td>
-          <td>{user.gender}</td>
-          <td>{atob(user.phone)}</td>
-          <td>{user.subject.subject}</td>
-          <td>{user.company.company}</td>
-          <td>{user.career.career}</td>
+          {
+            isEditable && userEdit.id === user?.id ? (
+              <td>
+                <FileUpload name='imageUrl' label='imageUrl' setFiledValue={
+                  (e, newValue) => {
+                    onFieldChange(user.id, 'imageUrl', newValue)
+                  }
+                }/>
+              </td>
+            ) : (
+              <td>
+                <div className="position-relative mx-auto" style={{width: '70px', height: '70px'}}>
+                  <img
+                    style={{objectFit: 'contain', width: '70px', height: '70px'}}
+                    alt={user.firstName}
+                    src={user.imageUrl}
+                  />
+                </div>
+              </td>
+            )
+          }
+          {
+            isEditable && userEdit.id === user?.id ? (
+              <td>
+                <input onChange={(e) => {
+                  onFieldChange(user?.id, 'firstName', e.target.value)
+                }} type="text" value={editedFields[user.id]?.firstName} placeholder={user?.firstName}/>
+              </td>
+            ) : (
+              <td>{user.firstName}</td>
+            )
+          }
+          {
+            isEditable && userEdit.id === user?.id ? (
+              <td>
+                <input onChange={
+                  (e) => {
+                    onFieldChange(user?.id, 'lastName', e.target.value)
+                  }
+                } type="text" value={editedFields[user.id]?.lastName} placeholder={user?.lastName}/>
+              </td>
+            ) : (
+              <td>{user.lastName}</td>
+            )
+          }
+          {
+            isEditable && userEdit.id === user?.id ? (
+              <td>
+                <input onChange={
```

Comment:
you are using material design in global scope, why you are using normal `input`, please change it for Material UI input